tkp/steps/forced_fitting.py

```diff
@@ -83,25 +83,27 @@ def perform_forced_fits(fit_posns, fit_ids,
         NB returned lists may be shorter than input lists
         if some fits are unsuccessful.
     """
-    logger.info("Forced fitting in image: %s" % (image_path))
-    fitsimage = tkp.accessors.open(image_path)
-
-    data_image = sourcefinder_image_from_accessor(fitsimage,
-                    margin=extraction_params['margin'],
-                    radius=extraction_params['extraction_radius_pix'],
-                    back_size_x=extraction_params['back_size_x'],
-                    back_size_y=extraction_params['back_size_y'])
-
-
-    boxsize = extraction_params['box_in_beampix'] * max(data_image.beam[0],
-                                             data_image.beam[1])
-    successful_fits, successful_ids = data_image.fit_fixed_positions(
-                                                fit_posns, boxsize, ids=fit_ids)
+    logger.info("Forced fitting in image: %s" % (accessor.url))
+
+    margin = extraction_params['margin']
+    radius = extraction_params['extraction_radius_pix']
+    back_size_x = extraction_params['back_size_x']
+    back_size_y = extraction_params['back_size_y']
+    data_image = sourcefinder_image_from_accessor(accessor, margin=margin,
```

Comment:
I just made it pep8 compatible while still being readable. personally I prefer having single action statement with clear names above complex nested calls with ambiguous spacing. But in the end that is up to the programmer to decide how to do it. In this case I really just wanted to remove the pep8 errors.